Phonebook.Frontend/src/app/modules/table/helpers.ts

```diff
@@ -94,6 +94,9 @@ export class Helpers {
    * @param str Any String
    */
   public static removeAccents(str: string): string {
+    if(str ==null){
```

Comment:
```suggestion
  public static removeAccents(str: string | null): string {
    if(str === null){
```
Bitte die Funktioneingabe dann auch anpassen

Reply:
Obwohl das eigentlich vorher behoben werden sollte.

Reply:
oder im Modell